### ISSUE TITLE
`server exited unexpectedly` when upgrading 3.3 -> 3.3a

### ISSUE THREAD

**Issue by SalvatoreT** [tmux/tmux#3219]
### Issue description

I just upgraded to 3.3a (via [Homebrew](https://brew.sh/)) and now I can't start a `tmux` session. When I try running it, I just get "server exited unexpectedly".

<img width="810" alt="Screen Shot 2022-06-09 at 5 17 48 PM" src="https://user-images.githubusercontent.com/1223008/172965900-eae464c7-dec0-47a3-ac30-bf236d7dc674.png">

### Required information

* tmux version: `tmux 3.3a`
* Platform: `Darwin arm`
* $TERM outside of tmux: `xterm-256color`
* Logs from tmux: none - it doesn't start
  ```
  ❯ tree /tmp/tmux-502
  /tmp/tmux-502
  └── default
  
  0 directories, 1 file
  ```
* Shell: `zsh 5.9 (arm-apple-darwin21.3.0)`

Please let me know what else I can tell you to help debug.

**Comment by krader1961** [comment]
Read issue #3203 where this has been mentioned several times. Likely causes are that you no longer have a plugin directory that your config expects to exist or otherwise have a `run-shell` command that is failing with a  non-zero exit status.

**Comment by SalvatoreT** [comment]
As a temporary fix, I uninstalled `tmux`, overwrote the 3.3a Hombrew formula, and re-installed the specific version that had `tmux` 3.3.

```
curl https://raw.githubusercontent.com/Homebrew/homebrew-core/6c1d6585dd6eca73b5735d5e51a18161815d3e2a/Formula/tmux.rb > $(find $(brew --repository) -name tmux.rb) && brew reinstall tmux
```

**Comment by SalvatoreT** [comment]
Ah, thanks for the tip, I'll take a look!

**Comment by SalvatoreT** [comment]
tl;dr For anyone else reading this, if there's an "a" in the `tmux` version, check to see if it's breaking your `bc`.

**Comment by SalvatoreT** [comment]
Thanks for your help, @krader1961!
